### DISCUSSION TITLE
A small mistake of the documentation.

### DISCUSSION THREAD

**Discussion by autra-tech-zhanghao** [coin-or/Ipopt#609] [General]
It seems that the col index is wrong of the second entry of the Hessian of the Lagrangian matrix in the [doc](https://coin-or.github.io/Ipopt/IMPL.html#TRIPLET).

**Comment by svigerske** [comment]
Yes, thank you. Fixed in 03b3eae0034c21d8e80c8cb1300e340e22ed75c7. Will be fixed in the online docs with the next release.
